cmd/controller-manager/app/options/options.go

```diff
@@ -109,4 +111,6 @@ func (o *Options) AddFlags(flags *pflag.FlagSet) {
 		"<group> for skip resources with a specific API group(e.g. networking.k8s.io),\n"+
 		"<group>/<version> for skip resources with a specific API version(e.g. networking.k8s.io/v1beta1),\n"+
 		"<group>/<version>/<kind>,<kind> for skip one or more specific resource(e.g. networking.k8s.io/v1beta1/Ingress,IngressClass) where the kinds are case-insensitive.")
+	flags.StringSliceVar(&o.SkippedPropagatingNamespaces, "skipped-propagating-namespaces", []string{},
+		"Comma-separated namespaces that should be skipped from propagating in addition to the default skipped namespaces(namespaces prefixed by kube- and karmada- and default).")
```

Comment:
We haven't skip the `default` namespace and have no plan on it by far.

Reply:
https://github.com/karmada-io/karmada/blob/master/pkg/controllers/namespace/namespace_sync_controller.go#L87

Reply:
It's the `namespace-sync-controller`'s default behavior, resource under `default` namespace still allowed to propagate.